.github/workflows/cpp-packaging.yml

```diff
@@ -39,26 +37,9 @@ env:
   xcodeVersion: "12"
```

Comment:
You can remove this env now.

Reply:
It's still needed, because it controls what xcode version the SDK is built with.

Reply:
The code  `sudo xcode-select -s /Applications/Xcode_${{ env.xcodeVersion }}.app/Contents/Developer` has been removed below.  Is there something else in here that uses it?  Some silent GHA observance of the variable?